### ISSUE TITLE
Simon Stevin data xCO2 plot not working properly

### ISSUE THREAD

**Issue by squaregoldfish** [quince-science/QuinCe#2908]
There's no Y axis. There is a range of data, so I'm not sure what's happening. Hiding the flags shows things as it should.

**Comment by squaregoldfish** [comment]
```
May  3 10:29:28 quince3 catalina.sh[221]: java.lang.NumberFormatException: For input string: "∞"
May  3 10:29:28 quince3 catalina.sh[221]: #011java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
May  3 10:29:28 quince3 catalina.sh[221]: #011java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
May  3 10:29:28 quince3 catalina.sh[221]: #011java.base/java.lang.Double.parseDouble(Double.java:651)
May  3 10:29:28 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.utils.MathUtils.nullableParseDouble(MathUtils.java:53)
May  3 10:29:28 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.web.datasets.plotPage.Plot.makePlotValues(Plot.java:223)
May  3 10:29:28 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.web.datasets.plotPage.Plot.init(Plot.java:267)
May  3 10:29:28 quince3 catalina.sh[221]: #011...
May  3 10:31:43 quince3 catalina.sh[221]: java.lang.NumberFormatException: For input string: "∞"
May  3 10:31:43 quince3 catalina.sh[221]: #011java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
May  3 10:31:43 quince3 catalina.sh[221]: #011java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
May  3 10:31:43 quince3 catalina.sh[221]: #011java.base/java.lang.Double.parseDouble(Double.java:651)
May  3 10:31:43 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.utils.MathUtils.nullableParseDouble(MathUtils.java:53)
May  3 10:31:43 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.web.datasets.plotPage.Plot.makePlotValues(Plot.java:223)
May  3 10:31:43 quince3 catalina.sh[221]: #011uk.ac.exeter.QuinCe.web.datasets.plotPage.Plot.init(Plot.java:267)
May  3 10:31:43 quince3 catalina.sh[221]: #011...
```

**Comment by squaregoldfish** [comment]
There's one or two values in the original data files of `1.7976931e+305` and `1.7976931e+308`. These seem to have been generated by a one-off issue, so we'll fix them manually (replacing with `-999.0` to match other missing values).
